src/util.ts

```diff
@@ -192,6 +199,32 @@ export function getRuntimeTrackingString(): string {
   }
 }
 
+export function getDirName() {
+  let dirToUse = '';
+  try {
+    dirToUse = __dirname;
+  } catch (e) {
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    // @ts-ignore
+    dirToUse = path.dirname(fileURLToPath(import.meta.url));
+  }
+
+  return dirToUse;
+}
+
+export function getPackageJSON() {
+  const dirName = getDirName();
+  return JSON.parse(
+    fs
+      .readFileSync(path.join(dirName, '..', '..', '..', 'package.json'))
```

Comment:
I _think_ this might be an issue with `esbuild`; perhaps we can have this as a separate .cjs file and import it from both module formats for now.

I considered suggesting we import this via:

```ts
await import('./package.json', {assert: {type: 'json'}});
```

However, it may produce 'experimental' warnings in some versions of Node.js.

Reply:
I think that import only works with higher versions of node. I remember bumping into this issue with 14. I'll try and dig up the exact version number.